### DISCUSSION TITLE
Google Docs Attachment Rendering

### DISCUSSION THREAD

**Discussion by adityakoolwal** [outline/outline#1661] [Self Hosting]
Hi! I just discovered this product and it is extremely close to what my company needs - we are *so* looking forward to getting off Confluence!

One of the hang-ups I anticipate running into is the rich attachment for Google docs. We often have tables that link to a number of different Google docs we've created, and the automatic rich attachment rendering every time you paste in a link to a doc would be frustrating for my team.

The preferred implementation would be to actually resolve to the Google doc title (similar to how you render a link to another Outline page), which the ability to toggle to a richer attachment if that's what you want.

Have you all thought about this? If you can get this AND ability to reorder pages in the Collection navigation, I think we'd be ready to switch over!

**Comment by tommoor** [comment]
Hey hey, this is fair feedback – it actually looks like embeds within tables don't even work 100% correctly even if you did want that functionality. I've filed an issue over on the editor repo: https://github.com/outline/rich-markdown-editor/issues/325

In the meantime you can get around this by typing text into a cell, selecting the text and then creating the link to the Google Doc that way. You can either make it a link through the toolbar or by pasting the link onto the selected text.

Hope this helps.

p.s. Reorderable sidebar is coming, just taking much more of a re-architecture than originally imagined.

**Reply by adityakoolwal** [reply]
Awesome - thanks for getting back to me! I’ll keep testing all the new features!
